osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmap.cs

```diff
@@ -102,37 +99,26 @@ private void load(BeatmapManager? manager, SongSelect? songSelect)
 
             Header.Children = new Drawable[]
             {
-                new Container
+                new BufferedContainer
                 {
                     RelativeSizeAxes = Axes.Both,
                     Children = new Drawable[]
                     {
-                        colourBox = new Box
-                        {
-                            RelativeSizeAxes = Axes.Y,
-                            Width = 40,
-                        },
-                        colourUnderline = new Box
+                        colourBox = new Box { RelativeSizeAxes = Axes.Both, },
```

Comment:
This starts as a very bright white before difficulty calculation kicks in or whatever. Probably should be something more neutral like a grey?

Reply:
right, and perhaps some easing im thinking?